PowerShellTools/DebugEngine/ScriptDebugger.cs

```diff
@@ -262,19 +284,27 @@ private void RefreshCallStack()
             IEnumerable<CallStack> result = null;
             try
             {
-                result = DebuggingService.GetCallStack();
+                if (IsDebugging)
+                {
+                    result = DebuggingService.GetCallStack();
+                }
+                else
+                {
+                    throw new Exception("Not in debugging mode any more!");

```

Comment:
We should throw a more specific exception if one exists; but if not, these can be bubbled up to the user and should be loc'd.